docs/content/Coding Milestones/PoC2023q1/where-resolver.md

```diff
@@ -107,6 +107,7 @@ I0605 10:53:00.261128   29786 controller.go:201] "starting controller" controlle
 Use workspace `root:compute` as the Inventory Management Workspace (IMW).
 ```shell
 kubectl ws root:compute
+kubectl kcp bind apiexport root:espw:edge.kcp.io
```

Comment:
This command does not belong here, unless I have overlooked something.

Reply:
probably a residual from desperate attempts to deal with the fundamental problem of cloning previously packaged release and not working from the current contribution. But I'm reluctant to fix what seems to work now,

Reply:
Yes, I tested without this addition and got as much success as with it.